### PR TITLE
Add dev and test commands to npm scripts

### PR DESCRIPTION
**Pull request by lukyanovas**

### CODE REVIEW

**Review by nightskylark — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -99,7 +99,9 @@
     "lint-staged": "lint-staged",
     "build": "dotnet build build/build-dotnet.sln && gulp default",
     "build-dist": "dotnet build build/build-dotnet.sln && gulp default --uglify",
-    "build-themes": "gulp style-compiler-themes"
+    "build-themes": "gulp style-compiler-themes",
+    "dev": "gulp dev",
+    "test": ".\\testing\\launch"
```
Comment (AlekseyMartynov) [inline package.json]: 🤷‍♂️ 
`test` isn't a task.
It launches UI and requires interaction.
I suggest that we gather more opinions.

Comment (AlekseyMartynov) [reply package.json]: Not cross-platform due to Win-style slashes.
`node testing/launch`?

Comment (nightskylark) [reply package.json]: Maybe rename this task to 'serve-test' or 'serve-qunit?

Comment (lukyanovas) [reply package.json]: Yes, I know, we can name it `test-launch` or something like that, but so many chars...

Comment (nightskylark) [reply package.json]: >so many chars...

How about `test-env`? :)

Comment (lukyanovas) [reply package.json]: `test-env` sounds good

**Review by AlekseyMartynov — APPROVED** [review]

**Pull request merged**